### PR TITLE
Fix Broken Link

### PR DESCRIPTION
**Pull request by mikeamoss**
Updated hyperlink to correct MS Docs page

### CODE REVIEW

File: WindowsServerDocs/identity/ad-fs/deployment/Best-Practices-Securing-AD-FS.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Protocol |Ports |Description
 HTTPS|443(TCP/UDP)|Used for device authentication.
 TCP|49443 (TCP)|Used for certificate authentication.
 
-For additional information on required ports and protocols required for hybrid deployments see the document [here](https://azure.microsoft.com/documentation/articles/active-directory-aadconnect-ports/).
+For additional information on required ports and protocols required for hybrid deployments see the document [here](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/reference-connect-ports).
```
Comment (ItzLevvie) [inline WindowsServerDocs/identity/ad-fs/deployment/Best-Practices-Securing-AD-FS.md]:
```suggestion
For additional information on required ports and protocols required for hybrid deployments see the document [here](https://docs.microsoft.com/azure/active-directory/hybrid/reference-connect-ports).
```

File: WindowsServerDocs/identity/ad-fs/deployment/Best-Practices-Securing-AD-FS.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Protocol |Ports |Description
...
 
 For detailed information about ports and protocols required for an Azure AD and Office 365 deployment, see the document [here](https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2?ui=en-US&rs=en-US&ad=US).
```
Comment (ItzLevvie) [inline WindowsServerDocs/identity/ad-fs/deployment/Best-Practices-Securing-AD-FS.md]:
```suggestion
For detailed information about ports and protocols required for an Azure AD and Office 365 deployment, see the document [here](https://docs.microsoft.com/office365/enterprise/urls-and-ip-address-ranges).
```

**Comment by illfated** [comment]
I concur with the suggested changes. Even if the pages are only available in English (en-us) at this time, it is usually a good idea to keep the links language agnostic, just in case some of them are (or will become) available in other languages.

**Comment by daveba** [comment]
@mikeamoss As @illfated stated, please add a commit that removes the `/en-us/` from the hyperlink.  We localize into the browser language.

**Comment by illfated** [comment]
Also @mikeamoss , if you are using an external git tool to manage your branches & content, you can make the necessary edits and add them to your PR branch. On the other hand, if that is not the case, please use the **[Commit changes]** feature within the suggestions above.

**Comment by eross-msft** [comment]
@ItzLevvie @mikeamoss  I believe since you added the change, you must use the **Commit suggestion** button for each of your suggestions. I can't merge anything until your suggestions are part of the actual PR. This must be done in the next 30 days or it will become necessary to close this PR.

**Comment by illfated** [comment]
@mikeamoss : It looks like you have not been active on GitHub since November 19, 2019.
Please keep us informed if you are available to follow up on the suggested changes.

**Comment by eross-msft** [comment]
@illfated  Closing this item. Please open a new PR so we can go ahead and accept these changes.

**Comment by illfated** [comment]
Very well, PR #3916 provided (as directly above).

**Comment by eross-msft** [comment]
@illfated  I believe all of these have now been merged. Please let me know if you notice anything missing or unmerged

**Comment by illfated** [comment]
Thanks, I just checked the content in **Update Link** **#3467**, the file [identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md) still contains 4 old-style technet links. Only 1 of them was corrected in PR #3467, so I need slightly more time than I spent on the other PRs to complete a working solution. Anyway, I will ping/CC you in the PR as soon as it is ready for review.

**Comment by illfated** [comment]
Oh and by the way: I may have mixed up the PRs #3466 and #3467 regarding content and the necessary work to complete the solution. Anyway, I will keep you posted.

**Comment by illfated** [comment]
Yes, I see it now. PR #3467 was resolved already when it got closed. I will look up and link the relevant commit(s) to #3467 to make it more readable for anyone who comes across that PR via web search or otherwise. That means only #3466 remains to resolve by providing a new PR.

**Pull request closed without merging**